frontend/server/src/main/java/org/pytorch/serve/openapi/OpenApiUtils.java

```diff
@@ -71,6 +71,22 @@ public static String getModelApi(Model model) {
         return JsonUtils.GSON_PRETTY.toJson(openApi);
     }
 
+    public static String getModelManagementApi(Model model) {
+        String modelName = model.getModelName();
+        OpenApi openApi = new OpenApi();
+        Info info = new Info();
+        info.setTitle("RESTful Management API for: " + modelName);
+        ConfigManager config = ConfigManager.getInstance();
+        info.setVersion(config.getProperty("version", null));
+        openApi.setInfo(info);
+
+        openApi.addPath("/models/{model_name}", getModelManagerPath(false));
+        openApi.addPath("/models/{model_name}/{model_version}", getModelManagerPath(true));
+        openApi.addPath("/models/{model_name}/{model_version}/set-default", getSetDefaultPath());
```

Comment:
listManagementApis can be applied here.

Reply:
listManagementApis has 6 paths:

/
/models
/models/{model_name}
/models/{model_name}/{model_version}
/models/{model_name}/{model_version}/set-default
/api-description

Here because it's only listing for specific model, it has only 3 paths:

/models/{model_name}
/models/{model_name}/{model_version}
/models/{model_name}/{model_version}/set-default